packages/playground/wordpress/src/wp-config.ts

```diff
@@ -0,0 +1,151 @@
+import { joinPaths, phpVars } from '@php-wasm/util';
+import type { UniversalPHP } from '@php-wasm/universal';
+
+/* @ts-ignore */
+import wpConfigTransformer from './wp-config-transformer.php?raw';
+
+/**
+ * Ensures that the "wp-config.php" file exists and required constants are defined.
+ *
+ * When a required constant is missing, it will be defined with a default value.
+ *
+ * @param php          The PHP instance.
+ * @param documentRoot The path to the document root.
+ */
+export async function ensureWpConfig(
+	php: UniversalPHP,
+	documentRoot: string,
+	defaultConstants: Record<string, string | number | boolean | null> = {}
+): Promise<void> {
+	const wpConfigPath = joinPaths(documentRoot, 'wp-config.php');
+
+	// The default values for constants listed in "wp-config-sample.php".
+	const defaults = {
+		DB_NAME: 'wordpress',
+		DB_USER: 'wordpress',
+		DB_PASSWORD: 'wordpress',
+		DB_HOST: 'localhost',
+		DB_CHARSET: 'utf8mb4',
+		DB_COLLATE: '',
+		AUTH_KEY: 'put your unique phrase here',
+		SECURE_AUTH_KEY: 'put your unique phrase here',
+		LOGGED_IN_KEY: 'put your unique phrase here',
+		NONCE_KEY: 'put your unique phrase here',
+		AUTH_SALT: 'put your unique phrase here',
+		SECURE_AUTH_SALT: 'put your unique phrase here',
+		LOGGED_IN_SALT: 'put your unique phrase here',
+		NONCE_SALT: 'put your unique phrase here',
```

Comment:
These could also get random values by default (Is manual WP install doing something like that? 🤔).